VSDiagnostics/VSDiagnostics/VSDiagnostics/Diagnostics/General/SingleEmptyConstructor/SingleEmptyConstructorAnalyzer.cs

```diff
@@ -65,6 +65,17 @@ private void AnalyzeSymbol(SyntaxNodeAnalysisContext context)
                 return;
             }
 
+            var classSymbol = context.SemanticModel.GetDeclaredSymbol(constructorDeclaration.Parent) as INamedTypeSymbol;
+            if (classSymbol != null && classSymbol.Constructors.Count() != 1)
+            {
+                return;
+            }
+
+            if (constructorDeclaration.GetLeadingTrivia().Any(t => t.IsKind(SyntaxKind.SingleLineDocumentationCommentTrivia)))
```

Comment:
Can you add a test that uses the `/* */` comment syntax before the constructor declaration? I would swap it with `Extensions.IsCommentTrivia()`